arwen/elrondapi/bigFloatOps.go

```diff
@@ -0,0 +1,893 @@
+package elrondapi
```

Comment:
I might suggest splitting each operation in its own file and putting them inside a folder for reducing the size of this.

Reply:
no splitting these are a lot of common things use here and there. common functions